### PR TITLE
BS-1061

### PR DESCRIPTION
**Pull request by DX-Bandwidth**
Update API specs from upstream api-specs repository. Opened By: @

### CODE REVIEW

File: site/src/pages/apis/index.js [hunk 1]
```diff
@@ -59,6 +59,11 @@ const apiList = [
         Svg: require('@site/static/img/api-icons/insights.svg').default,
         link: '/apis/insights/'
     },
+    {
+        title: 'Integrations API',
+        Svg: require('@site/static/img/api-icons/integrations.svg').default,
+        link: '/apis/integrations/'
+    },
```
Comment (habibmostafa) [inline site/src/pages/apis/index.js:66]: Who do I ask to design a new icon?

File: site/specs/messaging.yml [hunk 2]
```diff
@@ -1661,6 +1670,11 @@ components:
...
           nullable: true
           example: T
+        campaignId:
+          type: string
+          description: The campaign ID of the message if it has one.
+          nullable: true
+          example: CJEUMDK
     pageInfo:
       title: PageInfo
       type: object
```
Comment (habibmostafa) [inline site/specs/messaging.yml:1680]: Not sure how changes here came about.

File: site/static/img/api-icons/integrations.svg
Comment (habibmostafa) [inline site/static/img/api-icons/integrations.svg:1]: For this I just copied voice .... Who do I ask to design a new icon?

**Comment by DX-Bandwidth** [comment]
Corresponding Pull Request on [api-specs](https://github.com/Bandwidth/api-specs/pull/503) was closed.

**Comment by habibmostafa** [comment]
Opening BS-1132 to do xcaas instead of duets

**Pull request closed without merging**